exercises/word-count/examples/src/WordCount.purs

```diff
@@ -0,0 +1,32 @@
+module WordCount
+  ( wordCount
+  ) where
+
+import Control.Category ((>>>))
```

Comment:
I would just use `import Prelude` here and remove this import and the one for `Data.CommutativeRing`.